apps/frontend/package.json

```
@@ -26,7 +26,8 @@
 		"react-redux": "9.1.0",
 		"react-router-dom": "6.21.3",
 		"react-toastify": "10.0.4",
-		"shared": "*"
+		"shared": "*",
+		"vite-plugin-svgr": "4.2.0"
```

```suggestion
		"shared": "*"
```
We have this package in devDependencies already (Vlad changed it recently [commit])(https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-trackmates/commit/da822e6292963015a4c93b7cbfac6cce49d72dde)

---

right